04-gas/gas-section-seven/gas-section-seven.css

```diff
@@ -1,55 +1,146 @@
+    column-gap: var(--subcontainer-column-gap-desktop);
+    padding: var(--subcontainer-padding-desktop);
+  }
+}
+
+.gas-section-seven-copy-container {
+  grid-area: gas-section-seven-copy-container;
+  display: grid;
+  align-content: start;
+  row-gap: var(--copy-container-row-gap-mobile);
+  background-color: var(--level3-background-color);
+}
+
+@media (min-width: 600px) {
+  .gas-section-seven-copy-container {
+    align-content: center;
+    row-gap: var(--copy-container-row-gap-desktop);
+  }
+}
+
+.gas-section-seven-title-container{
+}
+
+@media (min-width: 600px){
+  .gas-section-seven-title-container{
+  }
+}
+
 .gas-section-seven-title {
-    grid-column: 1/2;
-    justify-self: center;
-    text-align: center;
-    font-size: var(--title-size-mobile);
-    /* width: var(--title-width-mobile);
-    height: var(--title-height-mobile); */
-    
-    background-color: var(--title-background-color-mobile);
-}
-
-.gas-section-seven-subtitle {
-    grid-column: 1/2;
-    justify-self: center;
-    text-align: center;
-    font-size: var(--title-size-mobile);
-    /* width: var(--title-width-mobile);
-    height: var(--title-height-mobile); */
-    
-    background-color: var(--title-background-color-mobile);
-}
-
-.gas-section-seven-content {
-    grid-column: 1/2;
-    text-align: justify;
-    font-size: var(--content-size-paragraph-mobile);
-}
-  
-.gas-section-seven-img {
-    display: block;
-    width: 100%;
-    border-radius: 30px;
-}
-  
-.gas-section-seven-img-container {
-    grid-column: 1/2;
-}
-  
\ No newline at end of file
+  display: flex;
+  flex-direction: column;
+  row-gap: var(--title-container-row-gap-mobile);
+  text-align: center;
+  font-size: var(--gas-title-font-size-mobile);
+  font-weight: var(--gas-title-font-weight-mobile);
+}
+
+@media (min-width: 600px) {
+  .gas-section-seven-title {
+    flex-direction: row;
+    column-gap: var(--title-container-column-gap-desktop);
+    row-gap: none;
+    text-align: left;
+    font-size: var(--gas-title-font-size-desktop);
+    font-weight: var(--gas-title-font-weight-desktop);
+  }
+}
+
+.gas-section-seven-title-span-number{
+}
+
+.gas-section-seven-title-span-title{
+}
+
+.gas-section-seven-paragraph-container {
+  display: grid;
+  row-gap: var(--paragraph-container-row-gap-mobile);
+}
+
+@media (min-width: 600px) {
+  .gas-section-seven-paragraph-container {
+  }
+}
+
+.gas-section-seven-paragraph {
+  text-align: center;
+  font-size: var(--paragraph-font-size-mobile);
+  font-weight: var(--paragraph-font-weight-mobile);
+}
+
+@media (min-width: 600px) {
+  .gas-section-seven-paragraph {
+    text-align: left;
+    font-size: var(--paragraph-font-size-desktop);
+    font-weight: var(--paragraph-font-weight-desktop);
+  }
+}
+
+.gas-section-seven-image-container {
+  grid-area: gas-section-seven-image-container;
+  display: grid;
+  align-content: start;
+  background-color: var(--level3-background-color);
+  justify-content: center; 
```

Comment:
as per previous PR
rearrange this in an order which makes sense hierarchically

justify-content (horiztonal positioning)
align-content (vertical positioning)
background-color (less important)